### DISCUSSION TITLE
Welcome to heos-controller Discussions!

### DISCUSSION THREAD

**Discussion by cold-logic** [cold-logic/heos-controller-electron#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thoxinhlalalavn-lab** [comment]
Hi, i download file install .exe for window. But it can not run with an error as image enclose. 
<img width="1920" height="1200" alt="Untitled" src="https://github.com/user-attachments/assets/191114f8-99d6-4de2-8a8b-47e373b65727" />
Please help
